src/main/scala/EnsimePlugin.scala

```diff
@@ -24,7 +24,7 @@ object EnsimeKeys {
     "Start up the ENSIME server and index your project."
   )
 
-  val ensimeServerVersion = settingKey[String](
+  val ensimeServerVersion = taskKey[String](
```

Comment:
But if that is the case, the user cannot override it at all, they will have to use SNAPSHOT.